src/workloads/tpch/denormalized/10/AvgAcctBal.yml

```diff
@@ -0,0 +1,19 @@
+SchemaVersion: 2018-07-01
```

Comment:
I only added these queries for scale 10, but it would be easy to add to scale 1 as well if we wanted to run them on that in future (or if we want to do that now, I can add them in this PR).